src/app/components/Article/__snapshots__/index.test.jsx.snap

```diff
@@ -9,14 +9,54 @@ Array [
   padding: 16px;
 }
 
+.c1 {
```

Comment:
@radiocontrolled out of interest which implementation/approach have you followed for this CSS? What we have is slightly different from what we use in Grandstand for the class 'gs-u-vh'. You mentioned webaim earlier, is this their approach?